setup.py

```diff
@@ -10,7 +10,7 @@
 
 MYDIR = path.abspath(os.path.dirname(__file__))
 
-VERSION = imp.load_source('version', path.join('.', 'falcon', 'version.py'))
+VERSION = importlib.import_module('falcon.version')
```

Comment:
This looks good, but just wondering, how is this semantically different from just `from falcon import version as VERSION`?

Do we need `importlib` at all?

Reply:
I'm using `importlib` because before it was using `imp`. I don't know if there are other differences